### ISSUE TITLE
JSX transform crashing on numeric and boolean literals inside braces

### ISSUE THREAD

**Issue by gaearon** [6to5/6to5#479]
I'm not sure if this is acorn-jsx bug or 6to5 bug.

```js
var x = <span>{3}</span>;
```

crashes the compiler:

```js
TypeError: undefined is not a function
    at exports.XJSElement.exit (/Users/dan/Documents/Projects/whatever/node_modules/6to5/lib/6to5/transformation/transformers/react.js:142:33)
```

This also happens for boolean literals inside `{}`.

**Comment by fkling** [comment]
Looks like a 6to5 bug. It refers to this line:

```js
var lines = child.value.split(/\r\n|\n|\r/);
```

But numbers and booleans don't have such a property.

**Comment by gaearon** [comment]
Thanks for quick fix.

**Comment by sebmck** [comment]
Released as of 2.12.0
